### PR TITLE
added forecast and downpayment percentage column to table 

### PR DESCRIPTION
**Pull request by harmlessprince**

### CODE REVIEW

File: src/views/Admin/ReportingTool.vue [hunk 1]
```diff
@@ -438,10 +441,10 @@ export default {
       }
     },
 
-	async getOrderTypes() {
+    async getOrderTypes() {
       try {
         const fetchOrderTypes = await get(this.apiUrls.orderTypes);
-		console.log(fetchOrderTypes);
+        console.log(fetchOrderTypes);
```
Comment (teebarg) [inline src/views/Admin/ReportingTool.vue:447]: remove console.log

File: src/views/Admin/ReportingTool.vue [hunk 2]
```diff
@@ -328,7 +328,9 @@ export default {
           this.query
         );
         this.reports = report.data.data;
-		this.branchesInfo = Object.values(this.reports.meta.groupedDataByBranch);
+        this.branchesInfo = Object.values(
+          this.reports.meta.groupedDataByBranch
+        );
         console.log("odododo");
```
Comment (teebarg) [inline src/views/Admin/ReportingTool.vue:334]: remove console.log

File: src/views/Admin/ReportingTool.vue (hunk 1 above)
Comment (harmlessprince) [reply src/views/Admin/ReportingTool.vue]: done

File: src/views/Admin/ReportingTool.vue (hunk 2 above)
Comment (harmlessprince) [reply src/views/Admin/ReportingTool.vue]: done

**Review by teebarg — APPROVED** [review]

**Pull request merged**